lib/src/Uncertainty/Model/DistributionImplementation.cxx

```diff
@@ -2458,6 +2458,15 @@ NumericalSample DistributionImplementation::getSupport() const
   return getSupport(getRange());
 }
 
+/* Get the discrete probability levels */
+NumericalPoint DistributionImplementation::getProbabilities() const
+{
+  if (!isDiscrete())
+    throw InternalException(HERE) << "Error: cannot return probability levels of a non discrete distribution.";
+
+  return computeCDF(getSupport()).getImplementation()->getData();
```

Comment:
-1 It is the PDF which gives the probability to take a given value for a discrete random vector, not the CDF